pkgs/applications/audio/muso/default.nix

```
@@ -0,0 +1,39 @@
+{ lib, fetchFromGitHub, rustPlatform
+, pkgconfig, wrapGAppsHook
+}:
+
+with rustPlatform;
+
+buildRustPackage rec {
+  pname = "muso";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "quebin31";
+    repo = "muso";
```

```suggestion
    repo = pname;
```

Not a blocker, more of a commonly done thing.